plugins/qtAccessionPlugin/lib/model/QubitAccession.php

```diff
@@ -26,10 +26,10 @@ public function __toString()
 
   protected function insert($connection = null)
   {
-    // If no identifier has been specified, use next available one
-    if (!$this->identifier)
+    // If identifier has been specified and the mask is enabled, increment the counter
+    if (!empty($this->identifier) && QubitAccession::maskEnabled())
```

Comment:
We could use `self` instead of `QubitAccession` like inside the `if`.

Reply:
Ah, true! I'll fix.